app/tests/spec/views/mixins/avatar-mixin.js

```diff
@@ -73,12 +70,24 @@ define([
       });
 
       it('deletes the url if null', function () {
-        view.updateAvatarUrl('url');
-        assert.isTrue(account.has('profileImageUrl'));
-        view.updateAvatarUrl(null);
-        assert.isFalse(account.has('profileImageUrl'));
-        assert.isTrue(user.setAccount.calledWith(account));
-        assert.isTrue(notifications.profileChanged.calledWith({ uid: UID }));
+        sinon.stub(account, 'fetchCurrentProfileImage', function () {
+          return p(new ProfileImage({ url: 'url', id: 'foo' }));
+        });
+        sinon.stub(account, 'deleteAvatar', function () {
```

Comment:
Not for this PR, but is there a difference between `Avatar` and `ProfileImage`? If not, should the names be unified?